packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -192,6 +193,8 @@ export const supportCopy = browser.isNative || document.queryCommandSupported('c
 // privileges to actually perform the action
 export const supportPaste = browser.isNative || (!browser.isChrome && document.queryCommandSupported('paste'));
 
+export const RECENT_COMMANDS_STORAGE_KEY = 'recent-commands';
```

Comment:
since you are saving `{ recent }` (line 238), i guess using 'commands' as the key would be more readable:

`path:commands: { recent: [ ] }`

instead of 

`path:recent-commands: { recent: [ ] }`

Reply:
sure, sounds fine :)